sandbox/create_admin.py

```diff
@@ -0,0 +1,32 @@
+import os
+from redisConnections.redis_users_connection import RedisUsersConnection
+
+
+if __name__ == '__main__':
```

Comment:
Preferably, put following code at the bottom of the file:
```
if __name__ == '__main__':
    main()
```
then put the functionality into `main()` method